### ISSUE TITLE
There are so many files present in a project from where to start finding bugs/solving bugs

### ISSUE THREAD

**Issue by sachinmukherjee** [OpenSourceHelpCommunity/Questions#12]
There are so many files present in a project from which file we should choose to start finding bugs or solving bugs?

**Comment by nikhita** [comment]
@sachinmukherjee The right way to choose an issue or bug is not through a file in the project but the "Issues" tab in Github. Pick an issue you want to work on (it's ok if you don't understand how to solve it, if it seems interesting to you, that's good enough!).

Once you choose an issue, _then_ look for the files this issue touches on. You don't need to concern yourself with _all_ files in the project. If you are having trouble understanding which files the issue deals with, comment on the issue asking this. Most of the times, the maintainer would be more than happy to point you out to the file in question. :)

**Comment by sachinmukherjee** [comment]
Thank you :)

**Comment by nikhita** [comment]
Closing this now. Please reopen if you have further questions on this. :)

**Comment by sachinmukherjee** [comment]
@nikhita can you please suggest me some small python based projects so that it will be easy for me to get started and gain some confidence.

**Comment by tapaswenipathak** [comment]
👋 @nikhita thank you for answering the question, it would be more helpful and easy to search if keep all the issues open all the time, only for this repository.

**Comment by nikhita** [comment]
@tapasweni-pathak ack, makes sense. 👍 

> can you please suggest me some small python based projects 

paging @vaibhavsingh97 :)

**Comment by vaibhavsingh97** [comment]
Hi @sachinmukherjee  There are plenty of python projects available and you can contribute any of them 😄 . I would suggest to look over the Google Summer Of Code organizations and see ehich org uses python for their projects and then select one of that org and start contributing to it.
P.S: Do have a look at @fossasia and @coala

**Comment by sachinmukherjee** [comment]
@vaibhavsingh97  sir not able to understand a single line of code of @coala project :-(

**Comment by vaibhavsingh97** [comment]
Oh! @sachinmukherjee No worries, So my first question would be that have you tried to install coala locally and also they had written extensive documentation for writing `coala-bears`, etc. So If you go through that documentation than probably you can't understand but reading for atleast 5 times will give you insight. So I would suggest share your problem with the community and ask for help they will guide with the process and you can ping us any time if you still face any difficulty. So always remember:
> Not asking a question is a crime 😉 

So ask questions and in OSS the best part is that nobody will say no and they will help you out 😄 as we guys here live like a family so don't be afraid while asking question from your family.
Happy Coding 👍

**Comment by sachinmukherjee** [comment]
@vaibhavsingh97 yes sir I have installed coala-bears and gone through documentation. what to do next sir
